src/TestInterfaces/TestInterfaces.csproj

```diff
@@ -0,0 +1,54 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need to add another project? Can't this be added to TestGrainInterfaces.csproj instead?